sssd_test_framework/utils/realmd.py

```diff
@@ -0,0 +1,101 @@
+"""Manage enrollment in realms."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.conn import ProcessResult
+
+__all__ = [
+    "RealmdUtils",
+]
+
+
+class RealmdUtils(MultihostUtility[MultihostHost]):
+    """
+    Call commands from realmd
+    """
+
+    def __init__(self, host: MultihostHost) -> None:
+        super().__init__(host)
+
+    def discover(self, args: list[Any] | None = None) -> ProcessResult:
+        """
+        Call ``realm discover``  with given arguments.
+
+        :param domain: Discover information about domains
+        :type domain: str,
+        :param args: additional arguments to pass to the discover operation
+        :type args: list,
+        """
+        if args is None:
+            args = []
+
+        return self.host.conn.exec(["realm", "discover", *args])
+
+    def leave(self, args: list[Any] | None = None) -> ProcessResult:
+        """
+        Call ``realm leave`` with given arguments.
+
+        :param args: additional arguments to pass to the leave operation
+        :type args: list,
+        """
+        if args is None:
+            args = []
+
+        return self.host.conn.exec(["realm", "leave", *args])
+
+    def join(
+        self,
+        domain: str,
+        args: list[Any] | None = None,
+        passwd: str | None = None,
+        user: str | None = None,
+        *,
+        krb: bool | None = None,
```

Comment:
All these should be keyword-only. Move the asterisk before args.

Reply:
done